docs/markdown/30-testing/01-others.md

```diff
@@ -13,4 +13,88 @@ If both cases, you can use **bats** to help you write a test suite for your acti
 
 # Testing Shell script with Bats
 
-TODO example
+## Test folder
+
+```yaml
+.github/
+    actions/
+        runs-using-docker/
+            cleanup.sh
+            main.sh
+            ...
+test/
+    bats/               <- submodule
+    test_helper/
+        bats-support/   <- submodule
+        bats-assert/    <- submodule
+    test.bats
+    ...
+
+```
+
+
+##--##
+
+# Testing Shell script with Bats
+
+## **test.bats** 
```

Comment:
```suggestion
**test.bats** 
```

Slide Theme don't like `##` for some reasons 😄

Reply:
done